bin/setup

```diff
@@ -14,7 +14,6 @@ chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
-  system! 'gem install bundler --conservative'
```

Comment:
I dont think we want to remove this line, in case someone setting up the app doesn't have bundler installed.

Reply:
As I understand it, and from my experience, bundler is now installed as part of the Ruby installation (I think this has been the case for a couple of minor Ruby versions or so.) Since the user is constrained to use our specified Ruby version, now 2.6, unless I'm mistaken, it is impossible for them to _not_ have bundler installed after they've installed the required version of Ruby.

Reply:
Ben and I agreed to leave this in, and add a version specification (now 1.17.2), because there may have been problems using Bundler v. 2.